website/pages/en/pptr.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const React = require('react')
```

Comment:
this was pretty much just copied from the others, I think that's what we're doing here? 🤷‍♂️

Reply:
Yeah, but a homepage is optional. You can link straight to the doc page from the root.